index.html

```diff
@@ -53,6 +53,169 @@ <h1>
 
                 </ul>
             </div>
+            <div class="action">
+                <img src="logo/Disabled.png" alt="">
+            </div>
+
+        </div>
+    </section>
+    <!-- Works -->
+    <section class="works">
+        <div class="works_content">
+            <h3>My Recent Works</h3>
+            <img src="logo/Indicator.png" alt="">
+        </div>
+        <div class="works_cards">
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Plaholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
```

Comment:
Great job on the card! However, there are a few improvements that can be made to align with the design and improve the code structure:

- [ ] It's recommended to use the `<article>` tag instead of `<div>` for the card element. This will ensure better semantic structure and enhance accessibility.

By implementing these changes, you will enhance the structure, semantics, and visual alignment of the cards in accordance with the design. Keep up the great work! 👍